### DISCUSSION TITLE
Export A Custom Trained Ultralytics YOLOv8 Model

### DISCUSSION THREAD

**Discussion by NLadi** [ultralytics/ultralytics#10103] [Show and tell]
Our new blogpost by [Nicolai Nielsen](https://github.com/niconielsen32) outlines how to train a custom Ultralytics YOLOv8 model, export the trained weights, and run live inference on a webcam.

🔎 Key Highlights:
✅ Training Custom Model 
✅ Exporting Model Weights 
✅ Live Inference on Webcams

Learn more ➡️ https://www.ultralytics.com/blog/export-a-custom-trained-ultralytics-yolov8-model

Join our GitHub Discussions to share your thoughts and learn more from the community!
[![Episode 4_ Export custom trained Ultralytics YOLOv8 model](https://github.com/ultralytics/ultralytics/assets/155453586/0c50c3e8-dd90-4977-a131-a0d5fdce01f7)](https://www.ultralytics.com/blog/export-a-custom-trained-ultralytics-yolov8-model)

**Comment by glenn-jocher** [comment]
Exciting times ahead for AI enthusiasts and developers! Nicolai Nielsen's latest blog post is a treasure trove for anyone looking to delve into custom AI models. 🌟 In his comprehensive guide, Nicolai walks us through training a custom Ultralytics YOLOv8 model, the nuances of exporting trained weights, and finally, setting up live inference directly from a webcam—a fantastic resource for practical, hands-on AI learning.

Whether you're a seasoned developer or just starting out, the insights on effectively harnessing the power of YOLOv8 are invaluable. 🚀 Don't miss out on this opportunity to enhance your skills and perhaps even kickstart your own innovative projects.

Dive into the full details and join the vibrant discussion on our GitHub page. Your thoughts and experiences can greatly contribute to and enrich the community's learning. See you there!

Check out the article here: [Export a Custom Trained Ultralytics YOLOv8 Model](https://www.ultralytics.com/blog/export-a-custom-trained-ultralytics-yolov8-model) 📖

Join the discussion and share your insights: [GitHub Discussions](https://github.com/orgs/ultralytics/discussions) 🗨️
